src/components/shared/cards/chart/index.css

```diff
@@ -0,0 +1,4 @@
+#card-wrapper {
+  flex-basis: 500px;
```

Comment:
Werkt dit ook met grotere schermen (vaste grote)?

Reply:
Is gefixed